From 40221c9d1dbfdb55fb692c20433095b7a3c3afd0 Mon Sep 17 00:00:00 2001
From: "shaikh.kamal" <shaikhkamal2012@gmail.com>
Date: Sun, 26 Jul 2026 12:13:09 +0530
Subject: [PATCH 07612/15333] docs: device-mapper: dm-inlinecrypt: fix 'bellow'
 spelling

Fix spelling error reported by codespell: bellow -> below.
No functional change.

Signed-off-by: shaikh.kamal <shaikhkamal2012@gmail.com>
Signed-off-by: Mikulas Patocka <mpatocka@redhat.com>
---
 Documentation/admin-guide/device-mapper/dm-inlinecrypt.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/admin-guide/device-mapper/dm-inlinecrypt.rst b/Documentation/admin-guide/device-mapper/dm-inlinecrypt.rst
index 76b3aae21eb4..8258a72f00a8 100644
--- a/Documentation/admin-guide/device-mapper/dm-inlinecrypt.rst
+++ b/Documentation/admin-guide/device-mapper/dm-inlinecrypt.rst
@@ -33,7 +33,7 @@ Parameters::
     or it can be passed as <key_string> prefixed with single colon
     character (':') for keys residing in kernel keyring service.
     You can only use key sizes that are valid for the selected cipher.
-    Note that the size in bytes of a valid key must be in bellow range.
+    Note that the size in bytes of a valid key must be in below range.
 
         [BLK_CRYPTO_KEY_TYPE_RAW, BLK_CRYPTO_KEY_TYPE_HW_WRAPPED]
 
-- 
2.55.0

